itest/rust/src/register_tests/constant_test.rs

```diff
@@ -40,15 +40,35 @@ impl HasConstants {
     #[constant]
     #[cfg(all())]
     const CONSTANT_RECOGNIZED_WITH_SIMPLE_PATH_ATTRIBUTE_BELOW_CONST_ATTR: bool = true;
+
+    #[constant]
+    const CFG_REMOVES_CONSTANT: i64 = 5;
+
+    #[cfg(any())]
+    #[constant]
+    const CFG_REMOVES_CONSTANT: i64 = 10;
+
+    #[constant]
+    #[cfg(any())]
+    const CFG_REMOVES_CONSTANT: i64 = 10;
+
+    #[cfg(any())]
+    #[constant]
+    const CFG_REMOVES_CONSTANT_FFI_GLUE: bool = true;
+
+    #[constant]
+    #[cfg(any())]
+    const CFG_REMOVES_CONSTANT_FFI_GLUE: bool = true;
```

Comment:
What is different here, compared to above? What does "FFI glue" mean? Note that tests should ideally be written against behavioral spec/constract, not implementation details.

Maybe choose a more expressive name and add a short description.

Reply:
I agree it's not very clear. Should probably be better now.